src/App.vue

```diff
@@ -32,13 +32,6 @@ export default {
 					vmid: 'referrer',
 					name: 'referrer',
 					content: 'strict-origin-when-cross-origin'
-				},
```

Comment:
Tests fail because it seems this information overlapped the meta description of the other pages, I will look at it to see if there is an overlapping or it's repeating the meta description tag